cases/management/commands/merge_entities.py

```
@@ -14,162 +14,166 @@
+    help = "Merge multiple entities into one, updating all references"
 
     def add_arguments(self, parser):
         parser.add_argument(
-            'entity_ids',
-            nargs='+',
-            type=int,
-            help='IDs of entities to merge (minimum 2 required)'
+            "entity_ids", nargs="+", type=int, help="IDs of entities to merge (minimum 2 required)"
         )
 
     def handle(self, *args, **options):
```

Ruff (0.15.2)

[warning] Unused method argument: `args`

(ARG002)